quic/s2n-quic-core/src/recovery/rtt_estimator.rs

```diff
@@ -118,25 +118,31 @@ impl RttEstimator {
     //# an acknowledgement of a sent packet.
     #[inline]
     pub fn pto_period(&self, pto_backoff: u32, space: PacketNumberSpace) -> Duration {
+        // Since K_GRANULARITY is 1ms, we operate on milliseconds rather than `Duration` to improve efficiency.
+        // See https://godbolt.org/z/4o71WPods
```

Comment:
Do you know how long these godbolt links remain valid?

Reply:
The one I added in varint is still valid :)

https://github.com/aws/s2n-quic/blob/1645cc5ee2f3fde753cf47d881bcdccd6648639d/quic/s2n-quic-core/src/varint/mod.rs#L45